### ISSUE TITLE
OS installer and A11Y support

### ISSUE THREAD

**Issue by celtichawk** [solus-project/os-installer#65]
This is something I've run into since 4.0 and 4.1 were released. Previously, in 3.9999999* the installer would be useable with the Orca screenreader. The desktops (Budgie, Gnome and Mate) are all useable with the built in screenreader.

However, the installer is not. I can set up a virtual machine, or create a bootable USB stick. I can boot up either Budgie, Gnome or Mate, and enable Orca. The moment I go into the installer, no events are passed to the A11y components at all. Orca does not read anything, it is not receiving any information at all, however scrolling up and down merely results in the sound for finding the top/bottom of the window. Can this be looked into, since this has been going on since 4.0 was released. If the installer is reverted to being Orca/Braille accessible it will give blind users anoher choice in terms of distros to work with.

**Comment by celtichawk** [comment]
As an update, Gnome is the sole installer that works with Orca. Mate and Budge and Plasma send no information to Orca or atspi at all. So is there any reason why this is the case, with Orca only working on one specific installer, yet working on Budgie, Gnome and Mate desktops?

**Comment by celtichawk** [comment]
As an update. Gnome is the sole installer that works with Orca. Budgie, Mate and Plasma send no events to the A11y packages at all, only Gnome is recieving and passing events through the screenreader
